src/main/java/com/java_essential/varycheva/essential/homework/hw1/task2/Book.java

```diff
@@ -0,0 +1,26 @@
+package com.java_essential.varycheva.essential.homework.hw1.task2;
+
+public class Book {
+
+    private Author author;
+    private Title title;
+    private Content content;
+
+    public void setAuthor(Author author) {
+        this.author = author;
+    }
+
+    public void setTitle(Title title) {
+        this.title = title;
+    }
+
+    public void setContent(Content content) {
+        this.content = content;
+    }
+
```

Comment:
At previous task you use constructor, why now use setters?

Reply:
I think that this is required by the task